lib/galaxy/jobs/__init__.py

```diff
@@ -1755,8 +1755,11 @@ def _change_ownership(self, username, gid):
             cmd.extend([self.working_directory, username, str(gid)])
             log.debug('(%s) Changing ownership of working directory with: %s' % (job.id, ' '.join(cmd)))
             p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            # TODO: log stdout/stderr
             stdout, stderr = p.communicate()
+            if p.returncode != 0:
+                logging.error('external script failed.')
+                logging.error('stdout was: %s' %stdout)
+                logging.error('stderr was: %s' %stderr)
```

Comment:
s/logging/log/ in the 3 lines above.

Also there's a missing space after the `%` operator in the 2 lines above.

Reply:
Hi Nick! Took me a few months before I could get our instance free enough to update it.

Good call - made a new commit.